liquibase-core/src/main/java/liquibase/sqlgenerator/core/DropProcedureGenerator.java

```diff
@@ -1,27 +1,46 @@
 package liquibase.sqlgenerator.core;
 
 import liquibase.database.Database;
+import liquibase.database.core.PostgresDatabase;
 import liquibase.exception.ValidationErrors;
 import liquibase.sql.Sql;
 import liquibase.sql.UnparsedSql;
 import liquibase.sqlgenerator.SqlGeneratorChain;
 import liquibase.statement.core.DropProcedureStatement;
 import liquibase.structure.core.Schema;
 import liquibase.structure.core.StoredProcedure;
+import liquibase.util.StringUtils;
+
+import static liquibase.structure.core.StoredProcedure.PROCEDURE_NAME_KEY;
 
 public class DropProcedureGenerator extends AbstractSqlGenerator<DropProcedureStatement> {
     @Override
     public ValidationErrors validate(DropProcedureStatement statement, Database database, SqlGeneratorChain sqlGeneratorChain) {
         ValidationErrors validationErrors = new ValidationErrors();
-        validationErrors.checkRequiredField("procedureName", statement.getProcedureName());
+        validationErrors.checkRequiredField(PROCEDURE_NAME_KEY, statement.getProcedureName());
```

Comment:
We validate Statement, but use constant from StoredObject which is inconsistent

Reply:
++